soap/tutorial-build-webservice-client.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: page
+key: tutorial-build-simple-webservice-client
+title: How to build a simple web service client
+---
+
+# 1. How to integrate web services in your own project.
+
+Go to the epages-soapclient repository at https://bintray.com/epages/maven/epages-soapclient/ and integrate it with your build automation tool or manually.
+
+## 1.1 Integrate it with your build automation tool.
+
+Include bintray to your maven repository resolution.
+For gradle this might look like this:
+
+{% highlight bash %}
```

Comment:
No, see also https://github.com/mkohl/epages-docs/blob/develop/soap/how-to-soapclient.md. Also this documentory is not for Github. It is shown here: https://developer.epages.com/soap/ with different markup.